File: 251001/Lashkin/src/Core/Domain/Entities/News.cs
namespace Domain.Entities;

public class News : BaseEntity
{
    public Guid UserId { get; set; }
    public string Title { get; set; } = null!;
    public string Content { get; set; } = null!;
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }
}

File: 251002/Berezun/publisher/src/schemas/news.py
from datetime import datetime

from pydantic import BaseModel, Field

class NewsBase(BaseModel):
    title: str = Field(min_length=2, max_length=64)
    content: str = Field(min_length=4, max_length=2048)
    creator_id: int = Field(alias="creatorId")

    model_config = {
        'from_attributes': True,
        'populate_by_name': True,
    }


class NewsRequestToAdd(NewsBase):
    pass

class NewsRequestToUpdate(NewsRequestToAdd):
    id: int

class NewsResponseTo(NewsBase):
    id: int
    created: datetime
    modified: datetime
